README.md

```diff
@@ -51,112 +51,65 @@ KubeSphere uses a loosely-coupled architecture that separates the [frontend](htt
 | Multi-tenant Management | Provide unified authentication with fine-grained roles and three-tier authorization system, supports AD/LDAP authentication |
 | Infrastructure Management | Support node management and monitoring, and supports adding new nodes for Kubernetes cluster |
-| Storage Support | Support GlusterFS, CephRBD, NFS, Local (default) etc. open source storage solutions, provide CSI plugins to consume storage from cloud providers |
-| Network Support | Support Calico, Flannel, etc. open source network solutions, provides load balancer plug-in [Porter](https://github.com/kubesphere/porter) for Kubernetes installed on physical machines |
+| Storage Support | Support GlusterFS, CephRBD, NFS, LocalPV (default), etc. open source storage solutions, provides CSI plugins to consume storage from cloud providers |
+| Network Support | Support Calico, Flannel, etc., provides [Network Policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) management, and load balancer plug-in [Porter](https://github.com/kubesphere/porter) for bare metal.|
 | GPU Support | Support add GPU node, support vGPU, enables running ML applications on Kubernetes, e.g. TensorFlow |
 
-Please See the [Feature and Benefits](https://kubesphere.io/docs/introduction/features/) for further information.
+Please see the [Feature and Benefits](https://kubesphere.io/docs/introduction/features/) for further information.
 
 ----
 
 ## Latest Release
 
-KubeSphere 2.1.1 was released on **February 23rd, 2020**. See the [Release Notes For 2.1.1](https://kubesphere.io/docs/release/release-v211/) for the updates.
+KubeSphere 3.0.0 is now generally available! See the [Release Notes For 3.0.0](https://kubesphere.io/docs/release/release-v300/) for the updates.
 
 ## Installation
 
 KubeSphere can run anywhere from on-premise datacenter to any cloud to edge. In addition, it can be deployed on any version-compatible running Kubernetes cluster.
 
-### Deploy on Existing Kubernetes Cluster
+### QuickStarts
 
-#### Prerequisites
+[Quickstarts](https://kubesphere.io/docs/quick-start/) include six hands-on lab exercises that help you quickly get started with KubeSphere.
 
-- `Kubernetes version`： `1.15.x, 1.16.x, 1.17.x`
-- `2.10.0 ≤ Helm Version ＜ 3.0.0` excluding 2.16.0 because of [#6894](https://github.com/helm/helm/issues/6894). Please see [Install and Configure Helm in Kubernetes](https://devopscube.com/install-configure-helm-kubernetes/). Helm v3 will be supported in KubeSphere 3.0.0.
-- An existing Storage Class in your Kubernetes cluster, use `kubectl get sc` to verify it
-- The CSR signing feature is activated in kube-apiserver, see [RKE installation issue](https://github.com/kubesphere/kubesphere/issues/1925#issuecomment-591698309).
+### Installing on Existing Kubernetes Cluster
 
-Install KubeSphere using kubectl.
+- [Installing KubeSphere on Amazone EKS](https://kubesphere.io/docs/installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-eks/)
```

Comment:
its better to point to the  specific version link since this URL will be changed to newer version.

Reply:
How about replacing the URL to `https://v3-0.docs.kubesphere.io/docs/`?

Reply:
yes that's what I am talking about.

Reply:
Done